src/components/TokenCard.js

```diff
@@ -126,7 +127,7 @@ function TokenCard(props) {
             <OptimizedImage src={TreeImg} width={104} height={104} />
           </Paper>
         </Grid>
-        <Grid container xs={8}>
+        <Grid container item xs={8}>
```

Comment:
So we can make a Grid both `container` and `item`?

Reply:
Yes, we can.